client/landing/gutenboarding/available-designs-config.json

```diff
@@ -48,7 +22,7 @@
 				"headings": "Playfair Display",
 				"base": "Fira Sans"
 			},
-			"categories": [ "featured", "blog" ],
+			"categories": ["featured", "blog"],
```

Comment:
FYI @Automattic/team-calypso in case this array whitespace change in json files is some unintended change in our Prettier fork.

Reply:
The fork works for me well locally. It's possible that the PR was formatted in some misconfigured environment, with the vanilla Prettier or a different config file.

Reply:
I think this is related to me trying out Nova https://nova.app/ -- so it likely was not configured correctly.

Reply:
Gotcha 👍  Just funky that "Check code style (Calypso)" test didn't fail but maybe Prettier doesn't run on json files?

Reply:
The "Check code style" CI job only runs ESLint on JS and TS files. `package.json` files are linted by `npm-package-json-lint`, but that tool doesn't check formatting: just the semantics of the content.

But the pre-commit hook does format staged `package.json` files. The fact that it didn't suggest that the commit was created with `git commit -n` 🙂

Reply:
I didn't use the `-n` flag, but maybe I did I tried the editor commit UI, I can't remember. Either way I'm using VSCode now, so no issue. :)